src/renderer/viz/expressions/reverse/ReverseArray.js

```diff
@@ -0,0 +1,42 @@
+import Base from '../base';
+
+/**
+ * Reverse the provided Array.
```

Comment:
I think this documentation should be merged with ReversePalette documentation to only have the documentation of `reverse`. Having two classes behind the scenes is an implementation detail

Reply:
Sounds good